sdk/search/azure-search-documents/azure/search/documents/_internal/_search_client.py

```diff
@@ -525,11 +532,41 @@ def index_documents(self, batch, **kwargs):
         :type batch: IndexDocumentsBatch
         :rtype:  List[IndexingResult]
         """
-        index_documents = IndexBatch(actions=batch.actions)
+        return self._index_documents_actions(actions=batch.actions, **kwargs)
```

Comment:
We should consider documenting what kind of exceptions this method can throw - which leads to the question "should the RequestEntityTooLarge" exception be public or not. And if not, should the type be introduced?

Reply:
Absolutely. I added this into APIView.